### PR TITLE
adding keypoints to sql lesson

### PR DESCRIPTION
**Pull request by wrightaprilm**

### CODE REVIEW

File: _episodes/09-working-with-sql.md [hunk 1]
```diff
@@ -3,14 +3,17 @@ title: Accessing SQLite Databases Using Python and Pandas
...
+    - "How can I write data from Python to be used with SQL?"
 objectives:
     - "Use the sqlite3 module to interact with a SQL database."
     - "Access data stored in SQLite using Python."
     - "Describe the difference in interacting with data stored as a CSV file versus in SQLite."
     - "Describe the benefits of accessing data using a database compared to a CSV file."
 keypoints:
-    - "FIXME"
+    - "sqlite3 provides a SQL-like interface to read, query, and write SQL databases from Python."
+    - "sqlite3 can be used with Pandas to read SQL data to the familiar Pandas DataFrame."
```
Comment (maxim-belkin) [inline _episodes/09-working-with-sql.md]: "Together, sqlite3 and Pandas can be used to read SQL tables into DataFrames"

File: _episodes/09-working-with-sql.md [hunk 2]
```diff
@@ -3,14 +3,17 @@ title: Accessing SQLite Databases Using Python and Pandas
...
+    - "Pandas and sqlite3 can also be used to transfer between the CSV and SQL formats."
```
Comment (maxim-belkin) [inline _episodes/09-working-with-sql.md]: Not sure about this keypoint... we usually transfer (something) between locations and convert data stored in one format to the other. I think I understand what you're trying to say but some clarification would help.

File: _episodes/09-working-with-sql.md [hunk 3]
```diff
@@ -3,14 +3,17 @@ title: Accessing SQLite Databases Using Python and Pandas
 teaching: 20
 exercises: 25
 questions:
-    - "FIXME"
+    - "What if my data are stored in an SQL database? Can I manage them with Python?"
```
[the remark this answers is not in the document]
Comment (wrightaprilm) [reply _episodes/09-working-with-sql.md]:
```suggestion
    - "Can I manage data stored in an SQL database with Python?"
```

**Comment by wrightaprilm** [comment]
Ooof, sorry about merging this, but I figured with the deadline coming, I ought to get us 90% of the way there. I don't think I can tackle these tonight, and I'm on deadline on two other projects. I'll PR these in ... sometime.

**Comment by maxim-belkin** [comment]
> Ooof, sorry about merging this, but I figured with the deadline coming, I ought to get us 90% of the way there. I don't think I can tackle these tonight, and I'm on deadline on two other projects. I'll PR these in ... sometime.

No worries. ✌️  I should've submitted them as I was typing them. I had to make a long break before I wrote the first comment and finished the last one.

**Pull request merged**